_posts/2017-09-26-testing-a-react-app.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: Testing a React application
+date: 2017-09-26
+---
+
+I spent the morning looking at ways to add tests to my React application. In this context I mean being able to test that 1) a React component renders without crashing; 2) it renders the correct DOM; 3) the business logic (embedded or not in a component) is correct.
+
+Browsing around, it seems that the stack [React](https://facebook.github.io/react/), [Jest](https://facebook.github.io/jest/) and [Enzyme](http://airbnb.io/enzyme/) is quite popular.
+Let's have a look at how it can help.
+
+**Disclaimer: if you came across that article, please bear in mind that I am nowhere an expert in these libraries so be very critical!**
```

Comment:
'nowhere _near_ an expert'?  

I think you can phrase this more positively, if you want, like 'I'm writing this to learn about these tools and I'm not an expert so feel free to submit comments or pull requests if you are and think something can be improved"

Reply:
Oops, missed that it should be :  'if you _come_ across _this_ article"

Reply:
thanks